src/quo2/components/switcher_tabs/constants.cljs

```diff
@@ -0,0 +1,8 @@
+(ns quo2.components.switcher-tabs.constants)
+
+(def switcher-header-tab-padding-horizontal 10)
+
+(def switcher-tabs-data [{:name "Switch" :color "#40444C"}
```

Comment:
move colors to quo2.colors namespace

Reply:
The colors will be removed eventually, once the real content is implemented. Should i still move these colors to `quo2.colors`?

Reply:
oh ok